### PR TITLE
sshpass: update to 1.10

### PR DESCRIPTION
**Pull request by manakbisht**
#### Description
Update `sshpass` version to `1.10`

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 14.2.1 23C71 arm64
Command Line Tools 15.1.0.0.1.1700200546

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: sysutils/sshpass/Portfile [hunk 1]
```diff
@@ -3,11 +3,11 @@
 PortSystem          1.0
 
 name                sshpass
-version             1.09
+version             1.10
 categories          sysutils
 license             GPL-2
 platforms           darwin
-maintainers         nomaintainer
+maintainers         @manakbisht
```
Comment (herbygillot) [inline sysutils/sshpass/Portfile:10]: Thanks for volunteering to be a maintainer for this port.  That said, a port should also be marked `openmaintainer` in general, which means that others besides you are allowed to make minor updates to this port.

```suggestion
maintainers         @manakbisht openmaintainer
```

File: sysutils/sshpass/Portfile [hunk 2]
```diff
@@ -30,6 +30,6 @@ long_description    SSH's (secure shell) most common authentication mode is \
 homepage            http://sourceforge.net/projects/sshpass/
```
Comment (herbygillot) [inline sysutils/sshpass/Portfile:30]: We should change all URLs to HTTPS while we're at it:

```suggestion
homepage            https://sourceforge.net/projects/sshpass/
```

**Comment by herbygillot** [comment]
Thanks for this submission, @manakbisht. I've added some comments above.

**Comment by manakbisht** [comment]
@herbygillot I have incorporated your feedback. Should I squash all these changes into a single commit?

**Comment by herbygillot** [comment]
> @herbygillot I have incorporated your feedback. Should I squash all these changes into a single commit?

Yes, that is the preferred policy. The commit has the exact same description as your original commit.

Thank you.

**Comment by herbygillot** [comment]
@manakbisht I have squashed the changes and committed through Github.

Thank you for your submission.

But yes, in the future, any additional modifications should be squashed back into the original commit. Thanks.

**Pull request merged**